cypress/e2e/api-tests/customers.cy.js

```diff
@@ -0,0 +1,135 @@
+describe('API /customers Test', () => {
```

Comment:
```suggestion
describe('API /customers', () => {
```

A palavra Test é redudante em um arquivo de teste.